experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -629,6 +633,105 @@ class _CheckboxRowState extends State<CheckboxRow> {
   }
 }
 
+enum Value { first, second }
+
+class Radios extends StatefulWidget {
+  const Radios({super.key});
+
+  @override
+  State<Radios> createState() => _RadiosState();
+}
+
+class _RadiosState extends State<Radios> {
+  Value? _value = Value.first;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: <Widget>[
+        Radio<Value>(
+          value: Value.first,
+          groupValue: _value,
+          onChanged: (value) {
+            setState(() {
+              _value = value;
+            });
+          },
+        ),
+        Radio<Value>(
+          value: Value.second,
+          groupValue: _value,
+          onChanged: (value) {
+            setState(() {
+              _value = value;
+            });
+          },
+        ),
+      ],
+    );
+  }
+}
+
+class ProgressIndicators extends StatefulWidget {
+  const ProgressIndicators({super.key});
+
+  @override
+  State<ProgressIndicators> createState() => _ProgressIndicatorsState();
+}
+
+class _ProgressIndicatorsState extends State<ProgressIndicators> {
+  bool showCircularProgressIndicator = false;
+  bool showLinearProgressIndicator = false;
+
+  Text buttonText(String text) {
+    return Text(
+      text,
+      style: const TextStyle(fontWeight: FontWeight.bold),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        TextButton(
```

Comment:
Not sure we really need these 'Hide/Show' buttons here, as we don't have them for any other components and they are taking up space.

We could also combine the circular and linear indicators into a single row to save on space as well.

Reply:
Thanks a lot for taking the time to review! 

I see. Since the indicators will animate all the time, I'm thinking that might bother the viewers🤣. So I'm just looking for a way to stop them. Do you have any suggestions?

This is what the indicators look like:
https://user-images.githubusercontent.com/36861262/192378734-d40c1ff7-7683-46af-806b-df2c4646e0ff.mov

Reply:
Ah, I forgot about the indeterminate state. Instead of a hide/show button for each, how about a Play/Pause icon button at the start of a row that has the circular and linear progress indicators. If it is paused we switch them from indeterminate to 70% complete. Does that make sense? That would allow someone to stop the animation if it was annoying them, but still show the indicators.

Reply:
Got it! Fixing:)
Thanks a lot!